commit 2abe385e9c68ec90716fef676a5b113721cc9c5f
Author: Rolf Eike Beer <kde@opensource.sf-tec.de>
Date:   Fri Sep 28 23:48:49 2012 +0200

    fix parallel build errors because of missing nie.h
    
    The interesting part of the CMake documentation is in the section about the
    OBJECT_DEPENDS property:
    
        This property need not be used to specify the dependency of a source
        file on a generated header file that it includes.  Although the
        property was originally introduced for this purpose, it is no longer
        necessary.  [...]  If the
        generated header file is created by another target, an inter-target
        dependency should be created with the add_dependencies command (if one
        does not already exist due to linking relationships).

diff --git a/autotests/lib/CMakeLists.txt b/autotests/lib/CMakeLists.txt
index f836dfd..0c2a963 100644
--- a/autotests/lib/CMakeLists.txt
+++ b/autotests/lib/CMakeLists.txt
@@ -34,6 +34,8 @@ kde4_add_library (nepomuktestlib
                   ${nepomuk_testlib_SRCS}
 )
 
+add_dependencies(nepomuktestlib nepomukcore)
+
 target_link_libraries (nepomuktestlib
                        ${QT_QTCORE_LIBRARY}
                        ${QT_QTDBUS_LIBRARY}
